### ISSUE TITLE
CoFH </3 Flaxbeard

### ISSUE THREAD

**Issue by karmarcharger** [Flaxbeard/Flaxbeards-Steam-Power#239]
COFH ore gen doesnt work with Flaxbeard installed

**Comment by elifoster** [comment]
I need more information than that.

**Comment by karmarcharger** [comment]
Eh...... I had custom ore gen with COFH.... Then I added Flaxbeard ( No config changes) ..... Ore gen didnt work in newly created worlds.... Removed Flaxbeard, ore gen works again....

**Comment by xbony2** [comment]
What versions are you using? Did anything funny go on in the logs?

**Comment by elifoster** [comment]
That isn't enough information, dude.

**Comment by karmarcharger** [comment]
Version 0.28.1...... No funny thing in the logs ;/... Because there wasnt anything in the logs it took me awhile to find it....

**Comment by karmarcharger** [comment]
@elifoster Ehh... Could you specify what info do you need? I wish I could provide a crash log... but it didnt crash... Do you want the server startup logs?

**Comment by karmarcharger** [comment]
http://paste.ee/p/dOyKO     Theres the logs

**Comment by elifoster** [comment]
I suppose a copy of your CoFH config, a copy of your FSP config, version numbers for everything involved. How does it not work?

**Comment by karmarcharger** [comment]
COFH Config file
http://pastebin.com/8vqPpUs5

Flaxbeard config
http://pastebin.com/mdEnNHVf

Version of Flaxbeard 0.28.1
Version of CoFHCore-[1.7.10]3.0.0RC5-207

**Comment by elifoster** [comment]
Just because there were some world gen changes since 0.28.1, I'd like you to test it with the latest 0.28.4. I doubt it'll do anything different, but it's worth a shot. How does it not work?

**Comment by karmarcharger** [comment]
Fixed in 0.28.4 O_o

**Comment by elifoster** [comment]
Probably has to do with:
```
world.provider.dimensionId = id;
```
to
```
if (id == world.provider.dimensionId)
```

from @c475856
